src/Aspire.Hosting.AWS/Utils/Internal/ProcessCommandService.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+using Microsoft.Extensions.Logging;
+using System.Diagnostics;
+using System.Text;
+
+namespace Aspire.Hosting.AWS.Utils.Internal;
```

Comment:
I didn't want to expose this interface as part of our public API but for unit testing purposes I need to implement the interface and making the interface internal doesn't work for that even marking the assembly internal visible to. So I went with the compromise of putting this in an "Internal" namespace.